implementations/rust/ockam/ockam_identity/src/secure_channel/options.rs

```diff
@@ -11,6 +11,12 @@ use core::fmt;
 use core::fmt::Formatter;
 use core::time::Duration;
 
+/// This is the interval before a credential expiration when we'll query for a new credential.
```

Comment:
I know it might be obvious to you but it could be good to add a comment about why we need this kind of parameter.
My guess is that we want to make sure that we don't end-up with an expired credential.